regression/expected/react-icu/TMExport_en_de_job_tmexport_en_de-req.json

```diff
@@ -304,6 +304,22 @@
 				" seconds"
 			],
 			"rid": "public/locales/en/app.json"
+		},
+		{
+			"sid": "nationalIdPlaceholder",
+			"guid": "6DAv7B6YywviZSG0LkH6iEmPaGfotuTy4Ljqm6eumfk",
+			"notes": {
+				"desc": "copy - national ID input placeholder on passenger form\nplaceholders: {\"id\":{\"example\":\"CPF\",\"description\":\"Name of a national ID\"}}"
+			},
+			"seq": 1006,
+			"nsrc": [
+				"Enter your ",
+				{
+					"t": "x",
+					"v": "{{id}}"
```

Comment:
The placeholder samples should be reflected in an `s` entry, for example [see](https://github.com/l10nmonster/l10nmonster/blob/49ad4ee6d1ec2ccb82dd362b827ac5b03821ca3d/tests/normalizers/util.test.js#L28)